src/Sulu/Bundle/TestBundle/Resources/app/config/sulu.yml

```diff
@@ -6,6 +6,7 @@ framework:
         enabled: false
     mailer:
         dsn: 'null://null'
+        message_bus: false # else messenger component duplicates the message: https://github.com/symfony/symfony/issues/48004#issuecomment-2476617713
```

Comment:
this seems to be a bug on symfony side which maybe fixed in the future at current state its easy to disable messenger integration in mailer for tests